packages/aws-cdk-lib/awslint.json

```diff
@@ -895,6 +895,9 @@
     "from-method:aws-cdk-lib.aws_apigatewayv2.HttpIntegration",
     "from-method:aws-cdk-lib.aws_apigatewayv2.HttpRoute",
     "from-method:aws-cdk-lib.aws_apigatewayv2.WebSocketIntegration",
-    "from-method:aws-cdk-lib.aws_apigatewayv2.WebSocketRoute"
+    "from-method:aws-cdk-lib.aws_apigatewayv2.WebSocketRoute",
+    "props-default-doc:aws-cdk-lib.aws_globalaccelerator.AcceleratorAttributes.dualStackDnsName",
+    "props-default-doc:aws-cdk-lib.aws_globalaccelerator.AcceleratorAttributes.ipv4Addresses",
+    "props-default-doc:aws-cdk-lib.aws_globalaccelerator.AcceleratorAttributes.ipv6Addresses"
```

Comment:
@msambol why were these added to awslint.json?

Reply:
I get a linter error if they're not in this file.

Reply:
The linter error happens when there is no`@default` value specified for the props.

Please include:
```
/**
 * @default <value>
 */
```

in the code comment for your props values and remove the changes to awslint.

Reply:
@sumupitchayan Pinged you on Slack. These are for imports, I think they'd all be `undefined` ?

Reply:
Updated.